client/src/components/notifications/MultipleUsersJoinPopover.vue

```diff
@@ -0,0 +1,93 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <ion-list class="users-list">
+    <ion-item
+      class="users-list-item body"
+      button
+      lines="none"
+      v-for="user in notificationData.roles"
+      :key="user.id"
+    >
+      <div class="users-list-item-content">
+        <user-avatar-name
+          :user-avatar="user.id"
+          :user-name="user.id"
```

Comment:
`UserID` is not the name of the user. You need the `HumanHandle` associated with this id.

Reply:
Updated ✔️